.ci/deps.sh

```diff
@@ -19,19 +21,21 @@ bundle install --path=vendor/bundle --binstubs=vendor/bin --jobs=8 --retry=3
 if ! dartanalyzer -v &> /dev/null; then
   wget -nc -O ~/dart-sdk.zip https://storage.googleapis.com/dart-archive/channels/stable/release/1.14.2/sdk/dartsdk-linux-x64-release.zip
   unzip -n ~/dart-sdk.zip -d ~/
+  cp -rp ~/dart-sdk/* ~/.local/
 fi
 
 # VHDL Bakalint Installation
-if [ ! -e ~/bakalint-0.4.0/bakalint.pl ]; then
+if [ ! -e ~/.local/bin/bakalint.pl ]; then
+  BAKALINT_VERSION=0.4.0
   wget "http://downloads.sourceforge.net/project/fpgalibre/bakalint/0.4.0/bakalint-0.4.0.tar.gz?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Ffpgalibre%2Ffiles%2Fbakalint%2F0.4.0%2F&ts=1461844926&use_mirror=netcologne" -O ~/bl.tar.gz
```

Comment:
What I have thought at https://github.com/coala/coala-bears/pull/2631 might look like this.

```
wget "http://downloads.sourceforge.net/project/fpgalibre/bakalint/$BAKALINT_VERSION/bakalint-$BAKALINT_VERSION.tar.gz"  -O ~/bl.tar.gz
```

Reply:
We dont need to change that line, so until we do, it can stay as it is.
e.g. if they release a new version, the variable would change, and the download link would use the variable.